src/TabContent.js

```diff
@@ -34,7 +34,7 @@ export default class TabContent extends Component {
     const classes = classnames('tab-content', this.props.className);
     return (
       <div className={classes}>
-        { this.props.children }
```

Comment:
For some reason it didn't throw me this error. Not sure if @eddywashere saw this when he release 3.1.0.